src/Core/Form/IdentifiableObject/CommandBuilder/Product/UpdateProductCommandsBuilder.php

```diff
@@ -70,12 +70,14 @@ public function buildCommands(ProductId $productId, array $formData, ShopConstra
             ->configureStockInformation($config, $formData)
         ;
 
-        $config->addField('[header][active]', 'setActive', DataField::TYPE_BOOL);
+        $config->addMultiShopField('[header][active]', 'setActive', DataField::TYPE_BOOL);
```

Comment:
active field was also forgotten to be handled for multiple shops here

Reply:
Nice catch!